src/contracts/pods/EigenPodManager.sol

```diff
@@ -273,31 +274,18 @@ contract EigenPodManager is
         address podOwner,
         uint256 prevRestakedBalanceWei,
         uint256 balanceDecreasedWei
-    ) internal returns (uint256, uint64, uint256) {
-        // Apply negative balance delta to calculate the proportion of the original
-        // balance that remains. Note that underflow here should be impossible given
-        // the invariants pods use to calculate these values.
+    ) internal returns (uint64) {
         uint256 newRestakedBalanceWei = prevRestakedBalanceWei - balanceDecreasedWei;
-        uint256 proportionRemainingWad = newRestakedBalanceWei.divWadRoundUp(prevRestakedBalanceWei);
-
-        // Note that underflow here should be impossible given
-        // `proportionRemainingWad` is guaranteed to be less than WAD.
-        uint256 wadSlashed = uint256(WAD) - proportionRemainingWad;
-
-        // Update pod owner's beacon chain slashing factor. Note that `newBeaconSlashingFactor`
-        // should be less than `prevBeaconSlashingFactor` because `proportionRemainingWad` is
-        // guaranteed to be less than WAD.
         uint64 prevBeaconSlashingFactor = beaconChainSlashingFactor(podOwner);
-        uint64 newBeaconSlashingFactor = uint64(prevBeaconSlashingFactor.mulWad(proportionRemainingWad));
-        emit BeaconChainSlashingFactorDecreased(podOwner, wadSlashed, newBeaconSlashingFactor);
-        /// forgefmt: disable-next-item
-        _beaconChainSlashingFactor[podOwner] = BeaconChainSlashingFactor({
-            slashingFactor: newBeaconSlashingFactor,
-            isSet: true
-        });
-
-        uint256 curDepositShares = uint256(podOwnerDepositShares[podOwner]);
-        return (curDepositShares, prevBeaconSlashingFactor, wadSlashed);
+        // newBeaconSlashingFactor is less than prevBeaconSlashingFactor because
+        // newRestakedBalanceWei < prevRestakedBalanceWei
+        uint64 newBeaconSlashingFactor =
+            uint64(prevBeaconSlashingFactor.mulDiv(newRestakedBalanceWei, prevRestakedBalanceWei));
+        uint64 beaconChainSlashingFactorDecrease = prevBeaconSlashingFactor - newBeaconSlashingFactor;
+        _beaconChainSlashingFactor[podOwner] =
+            BeaconChainSlashingFactor({slashingFactor: newBeaconSlashingFactor, isSet: true});
+        emit BeaconChainSlashingFactorDecreased(podOwner, beaconChainSlashingFactorDecrease, newBeaconSlashingFactor);
```

Comment:
this second parameter should be the `prevBeaconSlashingFactor`
```suggestion
        emit BeaconChainSlashingFactorDecreased(podOwner, prevBeaconSlashingFactor, newBeaconSlashingFactor);
```

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/936/commits/225e707681d42b55b9233cd2614b7440dc0cd09f